config/identity/handler/interaction/routes/forgot-password.json

```diff
@@ -4,15 +4,19 @@
     {
       "comment": "Handles all functionality on the forgot password page",
       "@id": "urn:solid-server:auth:password:ForgotPasswordRoute",
-      "@type": "InteractionRoute",
+      "@type": "BasicInteractionRoute",
       "route": "^/forgotpassword/?$",
       "viewTemplates": {
-        "InteractionRoute:_viewTemplates_key": "text/html",
-        "InteractionRoute:_viewTemplates_value": "@css:templates/identity/email-password/forgot-password.html.ejs"
+        "BasicInteractionRoute:_viewTemplates_key": "text/html",
+        "BasicInteractionRoute:_viewTemplates_value": "@css:templates/identity/email-password/forgot-password.html.ejs"
       },
       "responseTemplates": {
-        "InteractionRoute:_responseTemplates_key": "text/html",
-        "InteractionRoute:_responseTemplates_value": "@css:templates/identity/email-password/forgot-password-response.html.ejs"
+        "BasicInteractionRoute:_responseTemplates_key": "text/html",
+        "BasicInteractionRoute:_responseTemplates_value": "@css:templates/identity/email-password/forgot-password-response.html.ejs"
+      },
+      "controls": {
+        "BasicInteractionRoute:_controls_key": "forgotPassword",
+        "BasicInteractionRoute:_controls_value": "/forgotpassword"
```

Comment:
I don't like the leading slash here; it feels like an absolute path. Could we drop those?

Reply:
Should they also be dropped from the route regex then for consistency? That one is currently defined as `"route": "^/forgotpassword/?$"`

Reply:
Yes, I think so.

Reply:
I did the other changes, but this one I'm still thinking about. (I know approve means I can choose but I find it an interesting topic, feel free to say "do whatever" if you don't care :D ).

The idea with having the relative paths start with a slash is to have an element present in the regex/path that identifies the root, similar to how this is done in other places such as https://github.com/solid/community-server/blob/139edc6ceeb004280f5dd3e798b74755505e4d2e/config/http/static/default.json#L11

The main difference here is of course that the `/` does not reference the server root but the "idp root" (being `http://localhost:3000/idp/`). It mostly feels weird when changing the regex to `"^forgotpassword/?$"` since you still have the `^` token indicating the start of something.

On the other hand I might be biased by spending so much time in the code here, and by being a windows user (my file paths don't start with `/` ;) ).

Reply:
You're right, let's leave it as-is.